### PR TITLE
feat: add progress bar when downloading commands

### PR DESCRIPTION
**Pull request by wopol**
Issue: #7

### CODE REVIEW

**Comment by psmarcin** [comment]
Looks good but it takes into account only download process, what about extraction? It's not very common scenario to use huge file but we should cover that. Should it be included in the same progress bar as download or separate?

**Comment by wopol** [comment]
In my opinion extraction is not so important, because this is only one file and uncompress will be fast and stable operations unlike downloading. Downloading on the same computer can make take 2 seconds or two minutes depends on internet. Decompress 50mb file on my computer took 700 ms.

**Review by psmarcin — APPROVED** [review]

**Review by qzb — COMMENTED** [review]
Could you check [if output is a TTY](https://github.com/mattn/go-isatty) before initialising progress bar? I don't think we need these progress bars in CI pipeline logs.

**Comment by wopol** [comment]
No problem. Done

**Review by gpolek — APPROVED** [review]

**Review by psulowicz — APPROVED** [review]

File: internal/dependency/manager.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"os"
 	"path/filepath"
 
+	"golang.org/x/crypto/ssh/terminal"
```
Comment (bkielbasa) [inline internal/dependency/manager.go:13]: how about using golang.org/x/term instead? The package you use uses golang.org/x/term anyway but the dependency will be smaller

**Review by qzb — APPROVED** [review]

File: internal/dependency/manager.go (hunk 1 above)
Comment (wopol) [reply internal/dependency/manager.go:13]: Binary size when i use:
golang.org/x/term - 11273644
golang.org/x/crypto/ssh/terminal - 11273612

Comment (qzb) [reply internal/dependency/manager.go:13]: It does: https://github.com/golang/crypto/blob/master/ssh/terminal/terminal.go#L14

I guess `go mod download` is faster when you use golang.org/x/term directly.

**Pull request merged**